plugins/opercmds.py

```diff
@@ -204,3 +204,42 @@ def topic(irc, source, args):
     irc.call_hooks([irc.pseudoclient.uid, 'CHANCMDS_TOPIC',
                    {'channel': channel, 'text': topic, 'setter': source,
                     'parse_as': 'TOPIC'}])
+
+@utils.add_cmd
+def chghost(irc, source, args):
+    """<user> <new host>
+
+    Admin only. Changes the visible host of the target user."""
+    chgfield(irc, source, args, 'host')
```

Comment:
Just hardcode the field argument in uppercase and remove the `field.upper()` call later on